documentation/asciidoc/topics/yaml/cache_service.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: infinispan.org/v1
```

Comment:
I must have missed it. Where is this file and `datagrid_service.yaml` referenced?

Reply:
For both cache service and data grid service there are full yaml examples at: https://infinispan.org/docs/infinispan-operator/2.0.x/operator.html#ref_datagrid_service_crd-nodes 

I thought it would be useful to have a reference like this, especially downstream where https://github.com/infinispan/infinispan-operator/blob/master/doc/design.adoc is not available.